operator/tests/e2e-v2/decommission/00-create-redpanda.yaml

```diff
@@ -5,7 +5,8 @@ metadata:
   name: decommission
 spec:
   chartRef:
-    chartVersion: "5.7.9"
+    chartVersion: "5.9.9"
+    useFlux: false
```

Comment:
Did we decide to make a CLI flag that changes the default value of `useFlux` so we could run these cases against both fluxed and defluxed versions of the operator or am I making up a conversation? 😅

Reply:
You are right! I just wanted first fix all issuses that are orthogonal to cli flag. That's why this PR is based on Fixing artifact upload. My intention is to run test in CI as for me it's quicker to iterate. My local env is a little messy.